File: src/consumer/EventTriangleAPI.Consumer.BusinessLogic/Models/SupportTicketDto.cs
using EventTriangleAPI.Shared.DTO.Enums;

namespace EventTriangleAPI.Consumer.BusinessLogic.Models;

public record SupportTicketDto(
    Guid Id,
    Guid TransactionId,
    string UserId,
    Guid WalletId,
    string TicketReason,
    string TicketJustification,
    TicketStatus TicketStatus);

File: src/authorization/EventTriangle.Client/src/app/types/models/consumer/SupportTicketDto.ts
import {TicketStatus} from "../../enums/TicketStatus";

export interface SupportTicketDto {
  id: string;
  userId: string;
  wallerId: string;
  ticketReason: string;
  ticketJustification: string;
  ticketStatus: TicketStatus;
}
